integreat_cms/cms/views/region_condition/region_condition_view.py

```diff
@@ -70,3 +70,13 @@ def get(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
                 "search_query": query,
             },
         )
+
+    def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
+        r"""
+        Apply the query and filter the rendered regions
+        :param request: The current request
+        :param \*args: The supplied arguments
+        :param \**kwargs: The supplied keyword arguments
+        :return: The rendered template response
+        """
+        return self.get(request, *args, **kwargs, search_data=request.POST)
```

Comment:
Same as for contacts, I'd prefer that the search view would just use a GET request. But this is outside of the scope of this PR. Maybe a future refactoring target? Though that would likely be a lot of changes...